.github/workflows/upload-oci-artifacts.yaml

```diff
@@ -22,10 +25,50 @@ jobs:
         run: make
 
       - name: Upload OCI artifacts to GitHub packages
+        id: oci_build
         env:
           REGISTRY: ghcr.io
           REGISTRY_USER: ${{ github.repository_owner }}
           REGISTRY_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           REPO_GITHUB: https://github.com/${{ github.repository_owner }}/plugins.git
         working-directory: build/registry
-        run: ./bin/registry update-oci-registry ../../registry.yaml
+        run: >-
+          echo "REGISTRY_UPDATE_STATUS=$(
+            ./bin/registry update-oci-registry ../../registry.yaml
+          )" >> $GITHUB_OUTPUT
+
+      - name: Print registry update status
+        run: "echo ${{ steps.oci_build.outputs.REGISTRY_UPDATE_STATUS }}"
```

Comment:
leftover from testing? :)

Reply:
It can be cleaned up if you prefer :)
It could also be leveraged as verbosity.

Reply:
Yeah let's clean it up, also because it doesn't really work in the current form